jwala-persistence/src/main/java/com/cerner/jwala/persistence/service/impl/ResourceDaoImpl.java

```diff
@@ -244,6 +248,27 @@ public void updateResource(ResourceIdentifier resourceIdentifier, EntityType ent
         }
     }
 
+
+    @Override
+    public void updateResourceGroup(JpaApplication jpaApplication, JpaGroup jpaGroup) {
+        final Query q = em.createNamedQuery(JpaGroupAppConfigTemplate.UPDATE_RESOURCE_GROUP);
+
+        q.setParameter(JpaGroupAppConfigTemplate.QUERY_PARAM_APP, jpaApplication);
+        q.setParameter(JpaGroupAppConfigTemplate.QUERY_PARAM_GRP, jpaGroup);
+        int numEntitities = 0;
+        try {
+            numEntitities = q.executeUpdate();
+
+        } catch (RuntimeException re) {
```

Comment:
Don't ignore this RuntimeException.
I would add it as a Throwable to the ResourceTemplateUpdateException.